src/modules/Dashboard/components/CardImage/config.js

```diff
@@ -0,0 +1,17 @@
+import {
```

Comment:
Move variables from config to `src/modules/Dashboard/components/CardImage/CardImage.jsx`.